INSTALL.md

```diff
@@ -43,7 +43,7 @@ pip install https://github.com/biocore/mustached-octo-ironman/archive/master.zip
 pip install qiita-spots
 ```
 
-After these commands are executed, you will need (1) download a [sample Qiita configuration file](https://raw.githubusercontent.com/biocore/qiita/a0628e54aef85b1a064d40d57ca981aaf082a120/qiita_core/support_files/config_test.txt), (2) set the `QIITA_CONFIG_FP` environment variable and (3) proceed to initialize your environment:
+After these commands are executed, you will need (1) download a [sample Qiita configuration file](https://raw.githubusercontent.com/biocore/qiita/master/qiita_core/support_files/config_test.txt), (2) set the `QIITA_CONFIG_FP` environment variable and (3) proceed to initialize your environment:
```

Comment:
Do we need a hyperlink to the example config? If they pip install, they will already have the sample config. Or maybe the hyperlink is fine, but also mention that it's included in the pip intall